numpy/random/mtrand/mtrand.pyx

```diff
@@ -1040,7 +1040,7 @@ cdef class RandomState:
 
         Returns
         --------
-        samples : 1-D ndarray, shape (size,)
+        samples : int or ndarray
```

Comment:
int isn't really correct here - it's either an arbitrary scalar (`size == None`), or an `ndarray`